core/src/main/java/bisq/core/payment/payload/PaymentAccountPayload.java

```diff
@@ -129,6 +131,14 @@ public void setSalt(byte[] salt) {
         excludeFromJsonDataMap.put(SALT, Utilities.encodeToHex(salt));
     }
 
+    public String getHolderName() {
+        return excludeFromJsonDataMap.getOrDefault(HOLDER_NAME, "");
+    }
+
+    public void setHolderName(String holderName) {
+        excludeFromJsonDataMap.compute(HOLDER_NAME, (k, v) -> Strings.emptyToNull(holderName));
```

Comment:
Maybe a comment why to use compute here would be helpful (I see the reason, to ensure an empty string will result in the removal of the map entry as a null result in the mapping removes the entry).